app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
-    @posts = Kaminari.paginate_array(Post.all).page(params[:page]).per(5)
     @user = User.find(params[:user_id])
+    @posts = @posts = @user.posts.includes(:comments).page(params[:page]).per(5)
```

Comment:
- [ ] Kindly remove one `@posts = `. You should have something like this:

  ```
  @posts = @user.posts.includes(:comments).page(params[:page]).per(5)
  ```

Reply:
My mistake. I've fixed it now.